wp-content/themes/core/components/pagination/single/single.php

```diff
@@ -0,0 +1,35 @@
+<?php
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$c = \Tribe\Project\Templates\Components\pagination\single\Controller::factory( $args );
+
+?>
+<?php if ( ! empty( $c->previous_post ) || ! empty( $c->next_post ) ) { ?>
+	<nav <?php echo $c->wrapper_classes(); ?><?php echo $c->wrapper_attrs(); ?> >
+
+		<h3 <?php echo $c->header_attrs(); ?><?php echo $c->header_classes(); ?>>
```

Comment:
I kind of made this component up, do we want any tag customization on the header or the wrapper?

Reply:
Nope, and I'll actually nix the `h3` in favor of adding a `aria-label` and putting the static header text as the value